gtda/time_series/embedding.py

```diff
@@ -155,105 +195,71 @@ def resample(self, y, X=None):
+            parameter.
 
         Returns
         -------
         yr : ndarray of shape (n_samples_new,)
-            The resampled target. ``n_samples_new = (n_samples - time_delay *
-            (dimension - 1) - 1) // stride + 1``.
+            The resampled target. ``n_samples_new = (n_samples - size)
+            // stride + 1``.
 
         """
         check_is_fitted(self, '_is_fitted')
         yr = column_or_1d(y)
 
-        yr = np.flip(yr)
-        yr = np.flip(yr[:-self.width:self.stride])
+        yr = yr[:self.size - 2:-self.stride][::-1]
         return yr
 
-    @staticmethod
-    def plot(Xt, sample=0, plotly_params=None):
-        """Plot a sample from a collection of sliding windows, as a point
-        cloud in 2D or 3D. If points in the window have more than three
-        dimensions, only the first three are plotted.
-
-        Important: when using on the result `Xt` of calling :meth:`transform`
-        on ``X``, ensure that each sample in ``X`` is a point in
-        ``n_dimensions``-dimensional space with ``n_dimensions > 1``.
-
-        Parameters
-        ----------
-        Xt : ndarray of shape (n_samples, n_points, n_dimensions)
-            Collection of sliding windows, each containing ``n_points``
-            points in ``n_dimensions``-dimensional space, such as returned by
-            :meth:`transform`.
-
-        sample : int, optional, default: ``0``
-            Index of the sample in `Xt` to be plotted.
-
-        plotly_params : dict or None, optional, default: ``None``
-            Custom parameters to configure the plotly figure. Allowed keys are
-            ``"trace"`` and ``"layout"``, and the corresponding values should
-            be dictionaries containing keyword arguments as would be fed to the
-            :meth:`update_traces` and :meth:`update_layout` methods of
-            :class:`plotly.graph_objects.Figure`.
-
-        Returns
-        -------
-        fig : :class:`plotly.graph_objects.Figure` object
-            Plotly figure.
-
-        """
-        return plot_point_cloud(Xt[sample], plotly_params=plotly_params)
-
 
 @adapt_fit_transform_docs
-class TakensEmbedding(BaseEstimator, TransformerResamplerMixin):
-    """Representation of a univariate time series as a time series of
-    point clouds.
-
-    Based on a time-delay embedding technique named after F. Takens [1]_.
-    Given a discrete time series :math:`(X_0, X_1, \\ldots)` and a sequence
-    of evenly sampled times :math:`t_0, t_1, \\ldots`, one extracts a set
-    of :math:`d`-dimensional vectors of the form :math:`(X_{t_i}, X_{t_i +
-    \\tau}, \\ldots , X_{t_i + (d-1)\\tau})` for :math:`i = 0, 1, \\ldots`.
-    This set is called the :ref:`Takens embedding <takens_embedding>`
-    of the time series and can be interpreted as a point cloud.
+class SingleTakensEmbedding(BaseEstimator, TransformerResamplerMixin):
+    """Representation of a univariate time series as a time series of point
```

Comment:
I would stress the single by saying:
"Representation of a single univariate [...]"

Reply:
Thank you!

Reply:
Done. In fact the docstring on `master` was nonsense because it said "a time series of point clouds" which is not what this produces. A time series of points, perhaps. But I just wrote "a point cloud".